taskcluster/ci/config.yml

```diff
@@ -0,0 +1,27 @@
+---
+trust-domain: "mozilla"
+task-priority: low
+
+taskgraph:
+  cached-task-prefix: "mozilla.v2.perf.html"
```

Comment:
```suggestion
  cached-task-prefix: "mozilla.v2.profiler"
```

Reply:
thanks!
Yeah I wanted to generate the file again. My local directory is `perf.html` (the old name for this repository), that's why...

Reply:
You are welcome, I just noticed some more usage `perf.html` and highlighted it. The only usage with consequences outside of this repo is `cached-task-prefix` (it is used to index tasks in the [tasks/index](https://firefox-ci-tc.services.mozilla.com/tasks/index/mozilla.v2)). Other than that it looks good to me. I think if you land `.taskcluster.yml` on `main` you should see some tasks on this pull request. Using the local directory name is not ideal, so I filed https://github.com/mozilla-releng/reps/issues/10

Reply:
I put up a fix here:
https://github.com/taskcluster/taskgraph/pull/349

Though renaming the directory and re-generating is probably the fastest workaround :)